src/WebJobs.Script.WebHost/WebJobsServiceCollectionExtensions.cs

```diff
@@ -113,6 +114,7 @@ public static IServiceProvider AddWebJobsScriptHost(this IServiceCollection serv
             builder.Register<ISecretManager>(c => c.Resolve<WebHostResolver>().GetSecretManager()).ExternallyOwned();
             builder.RegisterType<WebFunctionsManager>().As<IWebFunctionsManager>().SingleInstance();
             builder.RegisterType<InstanceManager>().As<IInstanceManager>().SingleInstance();
+            builder.Register(_ => new HttpClient()).SingleInstance();
```

Comment:
using one instance of HttpClient. This is what we recommend to users, right?

Reply:
Yes